fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/IndexPredicate.java

```diff
@@ -440,8 +441,8 @@ public ValuePredicate(@Nonnull final List<String> fieldPath, @Nonnull final Inde
         }
 
         public ValuePredicate(@Nonnull final RecordMetaDataProto.ValuePredicate proto) {
-            Verify.verify(proto.getValueCount() > 0, String.format("attempt to deserialize %s without value", com.apple.foundationdb.record.query.plan.cascades.predicates.ValuePredicate.class));
-            Verify.verify(proto.hasComparison(), String.format("attempt to deserialize %s without comparison", com.apple.foundationdb.record.query.plan.cascades.predicates.ValuePredicate.class));
+            Verify.verify(proto.getValueCount() > 0, "attempt to deserialize " + ValuePredicate.class.getSimpleName() +  " without value");
+            Verify.verify(proto.hasComparison(), "attempt to deserialize " + ValuePredicate.class.getSimpleName() + " without comparison");
```

Comment:
Yeah... I don't particularly like either, if I'm honest. I think my preference would be to pass a `Supplier` in cases like this. I guess we'd want to do some kind of performance comparison, because string concatenation is pretty cheap, but the failure rate of this assert should also be pretty low. Maybe the real thing is we need to change this to not use `Verify`.

I've updated this to use the other `Verify` variant for now, though